src/inmanta/execute/runtime.py

```diff
@@ -20,6 +20,7 @@
 from typing import Deque, Dict, Generic, List, Optional, Set, TypeVar, Union, cast
 
 import inmanta.execute.dataflow as dataflow
+import inmanta.execute.proxy as proxy
```

Comment:
Shouldn't we add a changelog entry for this. This is a problem that existing in the release. No?

Reply:
True, but I assumed we would need to do a patch release for this and the changelog entry would added there and tied to the patch version. Should I add it here first, so we just have to move it around for the patch release?

Reply:
Yes, do add a changelog entry